content/kotsadm/updating/updating-kots-apps.md

```diff
@@ -6,16 +6,21 @@ weight: 10010
 draft: false
 ---
 
+## Using Admin Console
+
 The simplest way to update a KOTS application is through the "Version History" tab on the Admin Console.
+This method works for both online and airgapped installs.
 
-## Checking for Updates
-The Admin Console automatically checks for updates once every 15 minutes. 
-To manually check for a more recent version, click the "Check for updates" button at the top of the admin console. 
+### Checking for Updates
+The Admin Console automatically checks for updates once every 15 minutes.
+To manually check for a more recent version, click the "Check for updates" button at the top of the admin console.
+In airgapped instances this button will be replaced with a `Upload a new version` button.
+Airgapped instances cannot check for updates automatically.
 When an update is available, a new upstream version will show in the list of released versions.
```

Comment:
This is weird in this order. Sounds like "when an update is available" is directly related to airgapped instances.